src/commands/accounts/logOut.ts

```diff
@@ -1,12 +0,0 @@
-/*---------------------------------------------------------------------------------------------
-*  Copyright (c) Microsoft Corporation. All rights reserved.
-*  Licensed under the MIT License. See License.md in the project root for license information.
-*--------------------------------------------------------------------------------------------*/
-
-import { IActionContext } from '@microsoft/vscode-azext-utils';
-import { ext } from '../../extensionVariables';
-
-export async function logOut(_context: IActionContext): Promise<void> {
```

Comment:
Is the only way to logout through the VS Code accounts pane? It seems like it'd be nice to still support logging out via the command palette.

Reply:
That's correct--logging out is not possible programmatically, so the command has to be removed.